.github/actions/buildcache/action.yml

```diff
@@ -13,3 +13,4 @@ runs:
         go install github.com/buchgr/bazel-remote/v2@v2.4.1
         bazel-remote --azblob.tenant_id=bazelcache --azblob.storage_account=bazelcache --azblob.container_name=cache --azblob.auth_method=shared_key --dir=/tmp/bazelcache --max_size=2 --http_address=localhost:8080 &
         sleep 5 # Give the server time to start
+        echo $(curl ${{ inputs.cache_address }}/status)
```

Comment:
why do we need this? Should we just increase the sleep above?

Reply:
No, if you actually ping the status - it works but it says 0 files loaded. Even if you check status after an hour at the start of benchmarks, its 0 files. Must be auth issue or something - either way, the cache has broken several times and it completely stops the CI from working so this catches that. A separate PR should be used to fix the cache connection.

Reply:
ah, we need to update `BAZEL_REMOTE_AZBLOB_SHARED_KEY` then.